### PR TITLE
Issue191 ice options trickle

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -1403,6 +1403,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
               <t>An "a=msid-semantic:WMS" line MUST be added, as specified in
               <xref target="I-D.ietf-mmusic-msid"></xref>, Section 4.</t>
+
+              <t>An "a=ice-options" line, with the "trickle" option, as
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: I think you need to say "MUST be added" here; the format here is slightly different than the part below.

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -1886,8 +1885,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <t>The first step in generating an initial answer is to generate
           session-level attributes. The process here is identical to that
-          indicated in the Initial Offers section above.</t>
-
+          indicated in the Initial Offers section above, except
+          that the "a=ice-options" line, with the "trickle" option, as specified in
+          <xref target="I-D.ietf-mmusic-trickle-ice"></xref>, Section
+          4 is only included if such an option was present in the offer.</t>
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: add a comma after 4?

**Comment by juberti** [comment]
lgtm once comments are addressed

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -1403,6 +1403,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
               <t>An "a=msid-semantic:WMS" line MUST be added, as specified in
               <xref target="I-D.ietf-mmusic-msid"></xref>, Section 4.</t>
+
+              <t>An "a=ice-options" line, with the "trickle" option MUST be
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: I would remove the comma here

File: draft-ietf-rtcweb-jsep.xml [hunk 4]
```diff
@@ -1886,8 +1885,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <t>The first step in generating an initial answer is to generate
           session-level attributes. The process here is identical to that
-          indicated in the Initial Offers section above.</t>
-
+          indicated in the Initial Offers section above, except
+          that the "a=ice-options" line, with the "trickle" option, as specified in
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: I would remove the first comma (or the second)

**Comment by juberti** [comment]
lgtm with nonblocking comments

**Comment by fluffy** [comment]
LGTM

**Comment by ekr** [comment]
Relevant changes separately merged.

**Pull request closed without merging**